maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,90 @@
+
+Workshop data preparation (idr0021)
+===================================
+
+This document details the steps to prepare data from IDR for a workshop demonstrating
+analysis with Fiji, usage of Map Annotations and OMERO.tables and filtering with OMERO.parade.
+We use IDR0021, which is a Project containing 10 Datasets with a total of ~400 Images.
+
+
+Download IDR data
+=================
+
+External users
+--------------
+
+Please contact the OME team for access to download IDR data.
+
+Internal users
+--------------
+
+Follow instructions on the
+[IDR submission workflow](https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit#) to download the data via ```ssh``` from ```idr0-slot3```.
+
+
+Prepare IDR-metadata and import
+===============================
+
+Clone https://github.com/IDR/idr-metadata and edit
```

Comment:
Note that the IDR metadta repository is quite large due to the number of studies. Filtering out the `idr0021` study to its own submodule might help here if checkout time is an issue.